projects/super_res/model/autoreg_diffusion.py

```diff
@@ -1329,9 +1329,6 @@ def sample(self):
                     #videos, base, nsteps, flows = self.ema.ema_model.sample(lres, hres, True)
                     videos, base, nsteps, flows = self.ema.ema_model.sample(lres, hres)
 
-                    vmax = torch.max(torch.max(videos.max(), hres.max()), lres.max()).cpu().numpy()
```

Comment:
nit: I know you are removing this code, but I think you can use python's native `max` function here, e.g. 
```
max([videos.max(), hres.max(), lres.max()])
```
which I think would help clarify this line.

Reply:
aah okay